src/pipes/search.ts

```diff
@@ -1,11 +1,11 @@
-import * as Get from 'get-value';
 import { Pipe, PipeTransform } from '@angular/core';
+import * as Get from 'get-value';
```

Comment:
Through out the project we used capital letters for loaded modules, and I wanted it to stay consistent. It's a matter of opinion eventually, but consistency is more important I think